Update use-function-calling.mdx (#4416)
scaleway/docs-content: pages/generative-apis/how-to/use-function-calling.mdx

    tools=tools,
    tool_choice="auto"
)

print(response.choices[0].message.tool_calls)
```

> As the model detects properly that a tool call is required to answer the question, the output should be a list of tool calls specifying function names and parameters properties:

Reviewer: ```suggestion
As the model detects properly that a tool call is required to answer the question, the output should be a list of tool calls specifying function names and parameter properties:
```

    Some models must be told they can use external functions in the system prompt. If you do not provide a system prompt when using tools, Scaleway will automatically add one that works best for that specific model.
</Message>

> ### Call tool and provide final answer

Reviewer 2: ```suggestion
### Call the tool and provide a final answer
```

Function calling significantly extends the capabilities of language models by allowing them to interact with external tools and APIs. 

<Message type="note">

>   We cannot wait to see what you will build with function calls. Tell us what you are up to, and share your experiments on Scaleway's [Slack community](https://slack.scaleway.com/) #ai 

Reviewer 3: ```suggestion
  We cannot wait to see what you will build with function calls. Tell us what you are up to, and share your experiments on Scaleway's [Slack community](https://slack.scaleway.com/) in the #ai channel
```

<Message type="note">

>   We can't wait to see what you will build with function calls. Tell us what you are up to, share your experiments on Scaleway's [Slack community](https://slack.scaleway.com/) #ai 

Reviewer 2: ```suggestion
  We cannot wait to see what you will build with function calls. Tell us what you are up to, and share your experiments on Scaleway's [Slack community](https://slack.scaleway.com/) #ai 
```
